### ISSUE TITLE
Cannot reproduce the results.

### ISSUE THREAD

**Issue by gizi1** [qcraftai/distill-bev#3]
I ran the ./scripts/teacher_to_bevdepth4d/centerpoint2bevdepth.sh to train the student model, but the NDS  as follows was much lower than the results in the paper. I ran the training in 8 3080ti and set the batchsize from 64 to 32 for the limitation of GPU memory, which may affect the results.
2023-10-20 13:41:55,500 - mmdet - INFO - mAP: 0.3855
2023-10-20 13:41:55,501 - mmdet - INFO - mATE: 0.6319
2023-10-20 13:41:55,501 - mmdet - INFO - mASE: 0.7053
2023-10-20 13:41:55,501 - mmdet - INFO - mAOE: 1.5835
2023-10-20 13:41:55,502 - mmdet - INFO - mAVE: 0.3415
2023-10-20 13:41:55,502 - mmdet - INFO - mAAE: 0.1991
2023-10-20 13:41:55,502 - mmdet - INFO - NDS: 0.4050
2023-10-20 13:41:55,503 - mmdet - INFO - Eval time: 141.6s

I also evaluated the mAP and NDS of the teacher model (teacher_ckpts/centerpoint/epoch_20.pth) and the results seem also lower than the paper results. I got 0.487 mAP and 0.598 NDS. Is this right?

**Comment by dingwenli** [comment]
The learning rate and batch size may cause different results. 
Yes, that's right. The mAP and NDS of the pre-trained teacher model are 0.487 and 0.598 respectively. It is a modified CenterPoint (Pillar). We used it (teacher_ckpts/centerpoint/epoch_20.pth) to conduct distillation. FYI, the results in Table 2 come from the official CenterPoint (Voxel).

**Comment by Li-Whasaka** [comment]
Could you please tell me how long did it take you to train with 8 *3080ti？

**Comment by dingwenli** [comment]
We performed the centerpoint2bevdepth experiment on 8 * V100 and it took 49 hours.

**Comment by Chan-dh** [comment]
> I ran the ./scripts/teacher_to_bevdepth4d/centerpoint2bevdepth.sh to train the student model, but the NDS as follows was much lower than the results in the paper. I ran the training in 8 3080ti and set the batchsize from 64 to 32 for the limitation of GPU memory, which may affect the results. 2023-10-20 13:41:55,500 - mmdet - INFO - mAP: 0.3855 2023-10-20 13:41:55,501 - mmdet - INFO - mATE: 0.6319 2023-10-20 13:41:55,501 - mmdet - INFO - mASE: 0.7053 2023-10-20 13:41:55,501 - mmdet - INFO - mAOE: 1.5835 2023-10-20 13:41:55,502 - mmdet - INFO - mAVE: 0.3415 2023-10-20 13:41:55,502 - mmdet - INFO - mAAE: 0.1991 2023-10-20 13:41:55,502 - mmdet - INFO - NDS: 0.4050 2023-10-20 13:41:55,503 - mmdet - INFO - Eval time: 141.6s
> 
> I also evaluated the mAP and NDS of the teacher model (teacher_ckpts/centerpoint/epoch_20.pth) and the results seem also lower than the paper results. I got 0.487 mAP and 0.598 NDS. Is this right?

hi, I would like to ask you a question: When I tested using the official weights, the NDS and mAP were only 0.28 and 0.23. Have you encountered this situation before, and how did you solve it?
